src/tim/vx/ops/maxpoolwithargmax2_test.cc

```diff
@@ -138,11 +138,13 @@ TEST(MaxpoolGrad, without_overlay) {
                             out_shape, tim::vx::TensorAttribute::TRANSIENT);
     tim::vx::TensorSpec output_spec_values(tim::vx::DataType::FLOAT32,
                             out_shape, tim::vx::TensorAttribute::OUTPUT);
+    tim::vx::TensorSpec output_spec(tim::vx::DataType::FLOAT32,
+                            in_shape, tim::vx::TensorAttribute::OUTPUT);
 
     auto input_tensor = graph->CreateTensor(input_spec);
     auto output_tensor_indices = graph->CreateTensor(output_spec_indices);
     auto output_tensor_values = graph->CreateTensor(output_spec_values);
-    auto output_tensor = graph->CreateTensor(input_spec);
+    auto output_tensor = graph->CreateTensor(output_spec);
 
```

Comment:
why change this? single op_spec should be able to create multiply tensors!

Reply:
it's attr is wrong, create output tensor use the attr "INPUT"